reloader/resolver.py

```diff
@@ -11,13 +11,14 @@ def resolve_parents(root_name):
         that require that dependency, directly or indirectly.
         """
         manager = PackageManager()
-        everything = manager.list_packages() + manager.list_dependencies()
+        packages = manager.list_packages()
+        dependencies = map(lambda lib: lib.name, manager.list_libraries())
```

Comment:
Including libraries does no longer make sense as those are organized under $data/Libs/python{version}. They can not be reloaded like normal packages anymore.

Reply:
Thanks for pointing that out. I wasn't aware of it. Btw I've updated the code. This PR is just a simple fix to make it usable for now.